config/beaver_triple_service/client.sample.env

```diff
@@ -0,0 +1 @@
+BTS_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb29tX3V1aWQiOiIxMDUwOTMyYi0yODI2LTRmZDctOWNkMS04YzAyNjdhODc0ZmIiLCJwYXJ0eV9pZCI6MywicGFydHlfaW5mbyI6W3siaWQiOjEsImFkZHJlc3MiOiIxMC4wLjEuMjAifSx7ImlkIjoyLCJhZGRyZXNzIjoiMTAuMC4yLjIwIn0seyJpZCI6MywiYWRkcmVzcyI6IjEwLjAuMy4yMCJ9XSwic3ViIjoiYTk0NTgwNmUtNWIwMy00NmRhLWIxNDAtOWYxMDViMjFlMzJiIiwiZXhwIjo5MjIzMzcxOTc0NzE5MTc4NzUyfQ.qBXxgCR-2_tHysB1F_41pawvn7Y0NdxFikVuN2I3YRg
```

Comment:
This is just a comment.
The token BTS used seems better to use only registered claims and not carry any other information because this token is used for health check only.